homeassistant/components/radarr/const.py

```diff
@@ -0,0 +1,12 @@
+"""Constants for Radarr."""
+import logging
+
+from typing_extensions import Final
+
+DOMAIN: Final = "radarr"
+
+# Defaults
+DEFAULT_NAME = "Radarr"
+DEFAULT_URL = "http://172.17.0.1:7878"
```

Comment:
This default URL makes no sense. It's a local IP.

Reply:
Forgot that most people don't run supervised using that gateway address to access the docker default subnet.